src/loader.js

```diff
@@ -48,13 +53,9 @@ export default function (source) {
   logger.debug('Webpack config context:', webpackConfigContext);
   logger.debug('Resources:', resourcesLocations);
 
-  if (!resourcesLocations.length) {
-    const error = new Error(`
-      Something wrong with provided resources.
-      Make sure 'options.resources' is String or Array of Strings with a valid file path.
-    `);
-
-    return callback(error);
+  if (resourcesLocations.length === 0) {
+    logger.log('No sass resources files found from your config.');
```

Comment:
We should make the error message more complete to indicate allowed types and the fact they can be a glob.